types/src/user.rs

```diff
@@ -48,5 +54,78 @@ pub enum UserGroup {
 // // Impl FromStr and Display automatically for UserGroup, using serde.
 // // This allows us to (de)serialize lists of groups via a comma-separated string
 // // like this: "Editor,Contributor,Translator"
-// serde_plain::forward_from_str_to_serde!(UserGroup);
-// serde_plain::forward_display_to_serde!(UserGroup);
+serde_plain::forward_from_str_to_serde!(UserGroup);
+serde_plain::forward_display_to_serde!(UserGroup);
```

Comment:
@nole2701 in the future please avoid commenting out pieces of code! It looks like this was removed when you moved these declarations into the `types` codebase.